math/src/field/extensions/quadratic.rs

```diff
@@ -105,6 +105,10 @@ where
     fn from_base_type(x: [FieldElement<Q::BaseField>; 2]) -> [FieldElement<Q::BaseField>; 2] {
         x
     }
+
+    fn representative(_x: Self::BaseType) -> Self::BaseType {
+        todo!()
```

Comment:
[Same](https://github.com/lambdaclass/lambdaworks/pull/59/files#r1123112236) for this